canvas_modules/common-canvas/src/toolbar/toolbar.scss

```diff
@@ -179,7 +179,7 @@ $toolbar-divider-width: 1px;
 	& button {
 		padding: 0;
 		height: $toolbar-button-height;
-		min-height: 30px;
+		min-height: $spacing-07;
```

Comment:
revert since no matching token. Since there are multiple locations that uses `30px` as the height, can you move this to a variable?